misc/test/examples_test.go

```diff
@@ -370,7 +371,7 @@ func TestAccAzureJsWebserver(t *testing.T) {
 }
 
 func TestAccAzurePyAks(t *testing.T) {
-	t.Skip("Skipping due to ServicePrincipalNotFound error")
```

Comment:
Do we want to keep some record of this other reason this won’t work to make it easier to reenable other azure tests?